lib/utils/error-message.js

```diff
@@ -26,7 +26,7 @@ function errorMessage (er) {
     case 'EACCES':
     case 'EPERM':
       short.push(['', er])
-      detail.push(['', ['\nPlease try running this command again as root/Administrator.'
+      detail.push(['', ['\nPlease adjust file ownership or permissions -- or run this command again as a different user.'
```

Comment:
What about something along the lines of

> Please try adjusting file ownership or permissions, or running this as a user with root permissions.

Or maybe better yet, mentioning a couple other alternatives, like using a node version manager or creating a user dedicated directory.